src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java

```diff
@@ -71,10 +71,15 @@ public enum Mixins {
             () -> Hodgepodge.config.fixGlStateBugs,
             TargetedMod.VANILLA),
     FIX_FULLSCREEN_RESIZABLE(
-            "minecraft.MixinMinecraft",
+            "minecraft.MixinMinecraft_fixFullscreenResizable",
```

Comment:
I'm fine with either of prepending `fix` or not prepending `fix`, but not mixing them

Reply:
You are right, my mistake I copy pasted the names too fast. I'll remove the fix

Reply:
done